nimterop/docs.nim

```diff
@@ -36,7 +36,7 @@ proc execAction(cmd: string): string =
   doAssert ret == 0, "Command failed: " & $ret & "\ncmd: " & ccmd & "\nresult:\n" & result
 
 proc buildDocs*(files: openArray[string], path: string, baseDir = getProjectPath() & $DirSep,
-                defines: openArray[string] = @[]) =
+                defines: openArray[string] = @[], compilerArgs = "") =
```

Comment:
`nimArgs` maybe to disambiguate with C compiler? Do we really need this? What are examples of things to forward? `-d` perhaps?

Also, add to docs.

Reply:
`nimArgs` sounds good. Needed this to forward compiler flags from execTest tests when they specify flags for ast2